Library/Homebrew/cmd/fetch.rb

```diff
@@ -32,6 +32,11 @@ def fetch
   def fetch_bottle? f
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
+    if formula.file_modified?
+      ohai "#{formula.full_name}.rb is modified; fetching source"
```

Comment:
Maybe go for `PATH is modified!` and then putting the rest fitting on the next line? What do you think?